plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RepositoryCMAComponent.groovy

```diff
@@ -103,6 +104,7 @@ extends DirectComponentSupport
         name: input.name,
         type: input.type,
         format: input.format,
+        online: input.facet(ViewFacet).online,
```

Comment:
are we guaranteed to have a ViewFacet here? If there's doubt, might want to use ?. as a guard